docs/source/overview.rst

```diff
@@ -0,0 +1,81 @@
+.. _overview:
+
+================
+Package Overview
+================
+
+This overview should provide you with the basic insight on how Neural Monkey
+conceptualizes the problem of sequence-to-sequence learning how the data flow
+during training and look like.
+
+-------------------------------
+Loading and Processing Datasets
+-------------------------------
+
+We call a *dataset* a collection of named data *series*. By a series we a mean
+a sequence of data items of the same type representing one type of input or
```

Comment:
tady a všude jinde bych se vyvaroval používání slova sequence. Mluvíš o sequence-to-sequence learning, ale tadyto je, jak to jde po sobě v těch datech. Takže bych dal radši collection nebo list.